### ISSUE TITLE
Cannot Log in

### ISSUE THREAD

**Issue by Saborknight** [Aluxian/Facebook-Messenger-Desktop#281]
Hi,

I have just installed and re-installed with a fresh download. However, I have no idea where I am supposed to log in to Facebook... All I get is the attached screenshot. I've tried to forgot password and login using the web portal, and then refresh but it still thinks I'm not logged in. 

Dev tool is throwing the error attached.

Any help troubleshooting please???

Running a Windows 7 64x Ultimate

Saborknight
![screenshot_1](https://cloud.githubusercontent.com/assets/9272259/10884951/186cf66e-817b-11e5-8f75-2a23d1c0c2d7.png)
![screenshot_2](https://cloud.githubusercontent.com/assets/9272259/10884952/186f468a-817b-11e5-86c8-de0e4ad16c3c.png)
![screenshot_3](https://cloud.githubusercontent.com/assets/9272259/10884953/18c03af4-817b-11e5-8933-bca5cd2546c8.png)

**Comment by djsigfried56** [comment]
The error in the dev tools you're getting is from Facebook and not the tool itself. So the developer of this program can't do anything. Report this to Facebook you might get a reward. Check if the login page works in the browser: https://messenger.com

**Comment by Saborknight** [comment]
My Bad, forgot to update this issue. It was indeed the fault of Facebook, although in this case the cookie which it loaded with.

To Resolve:
I went into Dev Tools, deleted the existing cookie(s) (I only found the datr cookie) and reload.
What will happen is the fields will appear, however, once log in credentials have been entered, the app showed completely blank. At this point a simple reload and the app works perfectly now.

As I said, seems to be the cookies for some reason.

Anyway, thanks a lot!
